### PR TITLE
Add /customers/:id/documents endpoint to return all customer documents

### PR DESCRIPTION
**Pull request by louiseswift**
Includes a set of request tests and the config that creates & seeds a test database for these tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
     "debug": "export SLS_DEBUG=* && AWS_PROFILE=hackney node --inspect ./node_modules/serverless/bin/serverless offline -s dev --noAuth --noTimeout --port 3002",
     "auth": "serverless invoke local -s dev -f w2-document-api-authorizer -p authorizer/event.json",
     "lint": "eslint --fix --ext js api/lib && echo 'Lint complete.'"
+  },
+  "dependencies": {
```
Comment (bjpirt) [inline package.json:19]: Should this be in devDependencies?

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,14 +33,33 @@ jobs:
...
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12.13 && nvm alias default v12.13
+            npm i && pushd api && npm install && popd && pushd authorizer && npm install && popd
```
Comment (elena-vi) [inline .circleci/config.yml:52]: what does `popd` mean?

Comment (bjpirt) [reply .circleci/config.yml:52]: pushd - is like cd with a history (i.e. it pushes a directory on to the stack)
popd - changes your directory to where you were previously (i.e. it pops it off the stack)

Useful commands to know 👍

**Comment by louiseswift** [comment]
Added commit after feedback: https://github.com/LBHackney-IT/w2-document-api/pull/8/commits/1de552aa98076f23accaa25684d78ecd6f8c312c

There's now one combined SQL query for Academy-HB and Academy-CT documents, removing the need for a `ref` URL param 🙌 

The responsibility for processing Academy and UHW customer documents has been moved here, also, so Single View API does not need to worry about it (leading to a big clean up in the related PR: https://github.com/LBHackney-IT/housing-needs-single-view-proto-api/pull/61)

Happy to move the `processDocuments` definition if another file preferred.

Thank you for re-reviewing 🙏

**Pull request merged**